### PR TITLE
Add missing entries to the feature gates doc

### PR DESCRIPTION
**Pull request by ialidzhikov**
/area documentation

**Release note**:

```other operator
NONE
```

### CODE REVIEW

**Review by vpnachev — DISMISSED** [review]
/lgtm

File: docs/deployment/feature_gates.md [hunk 1]
```diff
@@ -86,10 +88,11 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
...
-* `HVPAForShootedSeed`  enables simultaneous horizontal and vertical scaling in managed seed (aka "shooted seed") clusters.
+* `HVPAForShootedSeed` enables simultaneous horizontal and vertical scaling in managed seed (aka "shooted seed") clusters.
 * `ManagedIstio` enables a Gardener-tailored [Istio](https://istio.io) in each Seed cluster. Disable this feature if Istio is already installed in the cluster. Istio is not automatically removed if this feature is disabled. See the [detailed documentation](../usage/istio.md) for more information.
 * `APIServerSNI` enables only one LoadBalancer to be used for every Shoot cluster API server in a Seed. Enable this feature when `ManagedIstio` is enabled or Istio is manually deployed in Seed cluster. See [GEP-8](../proposals/08-shoot-apiserver-via-sni.md) for more details.
 * `MountHostCADirectories` enables mounting common CA certificate directories in the Shoot API server pod that might be required for webhooks or OIDC.
+* `CachedRuntimeClients` enables a cache in the controller-runtime clients, that Gardener components use. If disabled all controller-runtime clients will directly talk to the API server instead of relying on a cache.
```
Comment (timebertt) [inline docs/deployment/feature_gates.md]: Should we add a note, that it is only available in gcm and gardenlet? https://github.com/gardener/gardener/pull/4320 removed the feature gate for g-scheduler.

**Review by timebertt — COMMENTED** [review]
Thank you!

**Review by timebertt — APPROVED** [review]
/lgtm

**Pull request merged**